### PR TITLE
Update rfxtrx.markdown to add info on adding ARC codes

### PR DESCRIPTION
**Pull request by krekr**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/rfxtrx.markdown [hunk 1]
```diff
@@ -392,7 +392,7 @@ If you need to generate codes for switches and lights, you can use a template (u
 
 {% endraw %}
 
-- Use this code to add a new switch in the options menu.
-- Launch your Home Assistant and go to the website.
+- Use this code to add a new switch in the options menu (Settings -> Devices -> RFXCOM -> Configure -> Paste the code into "Enter event code to add".
+- Launch your Home Assistant and go to the website or app.
 - Enable learning mode on your switch (i.e., push learn button or plug it in a wall socket)
-- Toggle your new switch in the Home Assistant interface
+- Toggle your new switch in the Home Assistant interface. Some brands (e.g. Trust / Kaku) use switching 'On' to learn, and turning 'Off' to unlearn, so make sure the switch is in the opposite state before enabling learning. E.g. if you want a device to learn the new code, make sure to toggle the switch in HA to 'Off' before starting the learning, then toggling the switch to 'On' while the learning mode is enabled on the device.
```
Comment (elupus) [inline source/_integrations/rfxtrx.markdown:398]: This make it somewhat unreadable. Place this as a note after the list of actions.

This is in the Lightwave section. This is a Lightwave type of brand?

Comment (krekr) [reply source/_integrations/rfxtrx.markdown:398]: Ah, I thought the list of actions was part of the 'Generate codes'-header.

Comment (krekr) [reply source/_integrations/rfxtrx.markdown:398]: feel free to edit and commit as you'd like, I'm lost in the github-commit-process :)
Must say that it's not the same as a wiki-edit ;)

**Comment by frenck** [comment]
Because there hasn't been any activity on this PR for quite some time now, I've decided to close it for being stale.

Feel free to re-open this PR when you are ready to pick up work on it again 👍

**Pull request closed without merging**